Sources/Testing/Testing.docc/Expectations.md

```diff
@@ -14,25 +14,40 @@ Check for expected values and outcomes in tests.
 
 ## Overview
 
-Tests may validate behaviors using expectations. This page describes the various
-built-in expectation APIs.
+The testing library provides `#expect()` and `#require()` macros you use to 
+validate expected outcomes and a `Confirmation` type you can use to confirm the 
```

Comment:
```suggestion
validate expected outcomes and a ``Confirmation`` type you can use to confirm the 
```

Reply:
Confirmations are sort of a "pro" feature and not something we'd want people to immediately reach for as their behaviour can be confusing especially when coming from `XCTestExpectation` and `XCTWaiter`.

Reply:
Yeah, let's focus on expect/require first, then lower down bring in the concept of confirmations.